config/routes.rb

```diff
@@ -2,4 +2,21 @@
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
  root 'homepage#index'
+ 
+  namespace :admin do
```

Comment:
i need this file fixed.

why do we have two resources block...what routes are generate by it.
and why do we have an admin controller why not user? as we already have admin namespace added?

Reply:
I have fixed it.
Sorry Sir.